frontend/javascripts/admin/voxelytics/workflow_view.tsx

```diff
@@ -373,8 +394,17 @@ export default function WorkflowView() {
     report == null || report.run.state === VoxelyticsRunState.RUNNING ? VX_POLLING_INTERVAL : null,
   );
 
+  if (loadingState.status === "FAILED" && user != null) {
```

Comment:
what happens if status is failed and user == null? this should probably show a log in form, right?

Reply:
the router shows a login form, if the user is not logged in. so effectively, `user == null` is never true for the scope of this component